where-to-travel/src/main/webapp/script.js

```diff
@@ -177,11 +194,13 @@ function showModal(htmlFilePath) {
     .then(content => openModal(content));
 }
 
-
+// When auth changes, display the proper dashboard
 firebase.auth().onAuthStateChanged(function(user) {
   $('#' + DASH_ID).empty();
   if (user) {
     addUserDash();
+    // When a user logs out, clear the saved plases set
```

Comment:
Clarify comment here since this doesn't seem to be the path for logged out users

Reply:
Fixed 80e3c5f